api/ResizeObserver.json

```diff
@@ -17,7 +17,7 @@
             "version_added": "69"
           },
           "firefox_android": {
-            "version_added": false
+            "version_added": "85"
```

Comment:
```suggestion
            "version_added": "79"
```

This is a relatively safe assumption to make, simply mirroring the Firefox data. Note that there was no Firefox 69-78 on Android, this is the first after 68.